src/crab/zone_domain.cpp

```
@@ -672,6 +672,27 @@ void ZoneDomain::forget(const VariableVector& variables) {
     normalize();
 }
 
+void ZoneDomain::rename(const std::vector<std::pair<Variable, Variable>>& renaming) {
```

Clang (14.0.6)

[warning] all parameters should be named in a function

(readability-named-parameter)